### ISSUE TITLE
[Master] Quest: Primal Strike (27027) offered

### ISSUE THREAD

**Issue by mmorganj** [TrinityCore/TrinityCore#18354]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

**Description:**
Primal strike incorrectly offered by Meela Dawnstrider (3062) in Camp Narache.
**Current behaviour:** (Tell us what happens.)
Meela Dawnstrider offers the player the quest Primal Strike to complete, but in legion, shaman players can't learn primal strike.
http://imgur.com/a/7wC1v
**Expected behaviour:** (Tell us what should happen instead.)
Meela Dawnstrider doesn't offer this quest to the player.
Source: http://www.wowhead.com/quest=27027/primal-strike
http://www.wowhead.com/quest=3093/rune-inscribed-note
**Steps to reproduce the problem:**

1. Create Tauren Shaman
2. Level to 3
3. Talk to Rohaku Stonehoof.
4. Begin Rune-inscribed Note (3093) - also is obsolete and shouldn't exist.
5. Complete 3093
6. Talk to Meela.
7. Obtain uncompletable quest - Primal Strike.

**Branch(es):** 3.3.5 / master (Tell us which branch(es) this issue affects.)
Master
**TC rev. hash/commit:**  
9b006e511bf40020e1660adf290ceb6ed6f4fd86
**TDB version:**  
703.00
**Operating system:**  
Windows 8.1

[//]: # (This template is for problem reports. For other types of report, edit it accordingly.)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)
[//]: # (For fixes containing C++, create a Pull Request.)

**Comment by tkrokli** [comment]
This issue could probably be solved simply by removing the NPCs from the QuestGiver & QuestEnder tables in the database, but to avoid linking errors and other issues, it needs to be done properly by making sure that spells or other parts of the source does not point to them. Does TDB 703.00 have a table for deprecated quests to make them stop working?

**Comment by nubqt** [comment]
@trokli: yes, I believe that by setting `QuestType` to '1' in `quest_template` that should disable the quest in-game.

The query would look like this:

```sql
UPDATE `quest_template` SET `QuestType` = '1' WHERE `id` = '27027';
```

I have not tested this though. Can someone please do?

**Comment by tkrokli** [comment]
OK, fair point. Then again, there is the `disables` table
https://trinitycore.atlassian.net/wiki/spaces/tc/pages/2130172/disables
where  `sourceType` = `DISABLE_TYPE_QUEST` (1):
```sql
SELECT * FROM `disables` WHERE `sourceType`= 1 AND `entry` BETWEEN 27000 AND 27200 ORDER BY `entry`;
```
```
sourceType   entry   flags  params_0  params_1  comment
----------  ------  ------  --------  --------  --------------------------------------------------------
         1   27018       0                      Deprecated quest: The Herald <NYI>
         1   27052       0                      Deprecated quest: Lingering Twilight
         1   27066       0                      Quest removed in patch 7.0.3: Learning the Word - Tauren
         1   27079       0                      Deprecated quest: Twilight Gate Tracking
         1   27080       0                      Deprecated quest: Twilight Gate Tracking
         1   27081       0                      Deprecated quest: [DEPRECATED]Trusting the Traitor
         1   27091       0                      Obsolete quest: Charge
         1   27121       0                      Deprecated quest: Free Knot!
         1   27122       0                      Deprecated quest: Fired Up and Not So Good To Go.
         1   27127       0                      Deprecated quest: Fired Up and Not So Good To Go.
         1   27137       0                      Deprecated quest: Earth Girls Aren't So Easy
         1   27138       0                      Deprecated quest: Earth Girls Aren't So Easy
         1   27150       0                      Deprecated quest: The Menders' Stead
```
... not sure why I didn't think of this table back in January.
